search.py

```diff
@@ -336,42 +340,44 @@ def extend(U, open_dir, open_other, g_dir, g_other, closed_dir):
         open_dir.remove(n)
         closed_dir.append(n)
 
-        for c in problem.actions(n):
-            if c in open_dir or c in closed_dir:
-                if g_dir[c] <= problem.path_cost(g_dir[n], n, None, c):
+        for child in problem.actions(n):
```

Comment:
I personally feel like `c` is descriptive enough, since in a lot of code c is used to denote the child.